doc/changes/latest.inc

```diff
@@ -24,6 +24,8 @@ Enhancements
 ~~~~~~~~~~~~
 .. - Add something cool (:gh:`9192` **by new contributor** |New Contributor|_)
 
+- Speed up :func:`mne.preprocessing.annotate_muscle_zscore`, :func:`mne.preprocessing.annotate_movement`, and :func:`mne.preprocessing.annotate_nan` through better annotation creation (:gh:`10089` **by new contributor** `Senwen Deng`_)
```

Comment:
Should be:

```
.. |Senwen Deng| replace:: **Senwen Deng**


- ... |Senwen Deng|_)
```
@drammock can you open a quick PR or push to `main` to fix it?

Reply:
oops. yes, will fix.